addons/website/static/src/snippets/s_popup/000.js

```diff
@@ -14,7 +15,9 @@ const PopupWidget = publicWidget.Widget.extend({
      * @override
      */
     start: function () {
-        this._bindPopup();
+        if (!utils.get_cookie(this.$target.attr('id'))) {
```

Comment:
Try not to use `$target` in widgets, it is meant to disappear some day. Use `this.$el` like for all widgets in Odoo.